lib/core/versions/latest/models/DidPublicKeyModel.ts

```diff
@@ -4,6 +4,7 @@
 export default interface DidPublicKeyModel {
   id: string;
   type: string;
+  controller: string;
```

Comment:
Design: I am contemplating if we should make this property optional so it can be used in both operation request, and resolution response.

Reply:
Leaving this the way it is as there's no need at this moment. The current tests are working because the input is cast'ed at runtime and the controller not present is not a problem.